extensions/amp-social-share/1.0/social-share.css.js

```diff
@@ -15,42 +15,37 @@
  */
 
 export const BASE_STYLE = {
-  'backgroundRepeat': 'no-repeat',
-  'backgroundPosition': 'center',
   'textDecoration': 'none',
   'cursor': 'pointer',
   'position': 'relative',
 };
 
 /* Twitter Styling */
-export const TWITTER = {'backgroundColor': '1da1f2'};
+export const TWITTER = {'color': 'ffffff', 'backgroundColor': '1da1f2'};
```

Comment:
What do you think about making this `DEFAULT_PRECONFIGURED_COLOR`? Looks like every single one has `ffffff`. Then later you can refer `CSS[type] && DEFAULT_PRECONFIGURED_COLOR`. Don't feel strongly about this though.

Reply:
Thanks for the recommendation, that's a good point.  I think for now I'd like to keep a `color` entry for each `type` so it feels like each `type` is unique from each of the other types (like they don't necessarily share the common `color`, but it just happens to be the same color for all types).  This may change in the future though if we change the design for the component more if we hear back from p counsel.